.chloggen/fixleak.yaml

```diff
@@ -0,0 +1,20 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'bug_fix'
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: exporterhelper
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Fix storage client leak in persistent queue when shutdown"
```

Comment:
The bug is not released yet, right? https://github.com/open-telemetry/opentelemetry-collector/pull/9090 was merged yesterday. So no need for a changelog entry